ci/plugins/cloudtest/hooks/command

```diff
@@ -22,6 +22,9 @@ fi
 
 date +"%Y-%m-%d %H:%M:%S" > step_start_timestamp
 
+ci_collapsed_heading ":docker: Purging all existing docker containers and volumes, regardless of origin"
+docker ps --all --quiet | xargs --no-run-if-empty docker rm --force --volumes
```

Comment:
Can --force cause any problems or is it fine since nothing is persisted anyway?

Reply:
I am hoping for the cleanest clean slate with which a job can start. So I think `--force` is appropriate here.